DataFormats/HcalDigi/src/classes_def.xml

```diff
@@ -42,11 +42,6 @@
    <class name="HcalTTPDigi" ClassVersion="10">
     <version ClassVersion="10" checksum="3127055275"/>
    </class>
-   <class name="HcalUpgradeDataFrame" ClassVersion="12">
-    <version ClassVersion="10" checksum="2143388425"/>
```

Comment:
Though I should note that HcalDigiProducer always produces all digi collections even if they're empty, so there could be empty HBHEUpgradeDigiCollections (or HF) in other samples. The relevant keep command in FEVTDEBUG is: `'keep *_simHcalUnsuppressedDigis_*_*'`. Will that cause a problem?